corehq/tabs/tabclasses.py

```diff
@@ -1139,7 +1139,7 @@ def contacts_urls(self):
     def settings_urls(self):
         settings_urls = []
 
-        if self.can_use_outbound_sms:
+        if self.can_use_outbound_sms and self.couch_user.is_domain_admin():
```

Comment:
I think you need this one level up. After this change user would see an empty "Settings" section. So you should add this check where setting_urls are being added in sidebar_items.

Reply:
oh i see General settings and languages added later, so this is going to get slightly less straightforward but we should not be showing the "Settings" section if there are no links under it.

Reply:
I have tested it on my local system and I remember that "Settings" tab was not displayed on Messaging page.
 I will verify it again.

Reply:
ah cool. Looks like it handles that later. thanks for confirming 

https://github.com/dimagi/commcare-hq/blob/05fa7f2bfd384e6aea2410e7c06e4e397e569bcf/corehq/tabs/uitab.py#L127